pkg/cli/cli.go

```diff
@@ -97,15 +96,22 @@ var (
     To see the full list of commands supported, run 'oc --help'.`)
 )
 
-func NewDefaultOcCommand(in io.Reader, out, errout io.Writer) *cobra.Command {
-	cmd := NewOcCommand(in, out, errout)
+func defaultConfigFlags() *genericclioptions.ConfigFlags {
+	return genericclioptions.NewConfigFlags(true).WithDeprecatedPasswordFlag().WithDiscoveryBurst(350).WithDiscoveryQPS(50.0)
+}
+
+func NewDefaultOcCommand(o kubecmd.KubectlOptions) *cobra.Command {
```

Comment:
This is breaking interface change. Thereby,  I searched external usage of this function and there is none found which means that this change is safe.